src/metatensor/models/cli/train.py

```diff
@@ -206,15 +206,36 @@ def _train_model_hydra(options: DictConfig) -> None:
     :param options: A dictionary-like object obtained from Hydra, containing all the
         necessary options for dataset preparation, model hyperparameters, and training.
     """
+
+    architecture_name = options["architecture"]["name"]
+    architecture = importlib.import_module(f"metatensor.models.{architecture_name}")
+    architecture_capbilities = architecture.__ARCHITECTURE_CAPABILITIES__
```

Comment:
```suggestion
    architecture_capabilities = architecture.__ARCHITECTURE_CAPABILITIES__
```
Here and everywhere below this

Reply:
thanks!